azuredeploy.json

```diff
@@ -31,13 +31,13 @@
     "localAdminUsername": {
       "type": "string",
       "metadata": {
-        "description": "The name of the local admin account for connection service VM's"
+        "description": "The name of the local admin account for connection service VM's and the domain controller, if created"
```

Comment:
Should VM be Remote Workstations?

Reply:
In this case not - it's the local admin password for the security gateways, connection server, and domain controller (if created).